src/reset-password/ResetPasswordPage.jsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button, Input, ValidationFormGroup } from '@edx/paragon';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+
+import messages from './messages';
+import { resetPassword, validateToken } from './data/actions';
+import { resetPasswordResultSelector } from './data/selectors';
+import InvalidTokenMessage from './InvalidToken';
+import ResetSuccessMessage from './ResetSuccess';
+
+const Spinner = () => (
+  <div className="d-flex justify-content-center reset-password-container">
+    <div className="d-flex flex-column" style={{ width: '400px' }}>
+      <div className="form-group">
+        <div className="d-flex flex-column align-items-start">
+          Validating Token
```

Comment:
Remove the text and spinner should be in aligned center both vertically and horizontally.